torchci/lib/bot/autoLabelBot.ts

```diff
@@ -113,9 +115,11 @@ const notUserFacingPatternExceptions: RegExp[] = [
 // ]
 const repoSpecificAutoLabels: {[repo: string]: [RegExp, string][]}  = {
   // Sample entry below
-  // "pytorch/pytorch": [
-  //    [/torch\/(csrc|cpp)\/api/gi, "release notes: cpp"]
-  // ],
+  "pytorch/pytorch": [
+      [/aten\/src\/ATen\/mps/gi, , "ciflow/mps"],
```

Comment:
Extra comma causing errors here
```suggestion
      [/aten\/src\/ATen\/mps/gi, "ciflow/mps"],
```